### DISCUSSION TITLE
@twilio-paste/core 4.1.9 not working below Webpack 4.30.0

### DISCUSSION THREAD

**Discussion by GiantRobots** [twilio-labs/paste#949] [Q&A] [answered]
For some reason with the current version of paste-core `4.1.9` using a Webpack version before `4.30` will cause an error `Uncaught TypeError: Object(...)(...) is not a function`. 

You can observe this in the simple reproduction repo by running `npx webpack serve`. To see the error disappear upgrade webpack from `4.29.6` to `4.30.0` and re-run the serve command.

Not 100% sure why this is happening but based on the webpack [4.30.0 release notes ](https://github.com/webpack/webpack/releases/tag/v4.30.0) I think it's probably related to one of the bug fixes in this release.

> * fix generating a null id when concatenating an entry module in a different chunk
> * fix parsing of function name in function scope

[pasteTest.tar.gz](https://github.com/twilio-labs/paste/files/5555765/pasteTest.tar.gz)

**Comment by GiantRobots** [comment, marked as answer]
Upgrade to `webpack@^4.30.0`

**Reply by TheSisb** [reply]
Thanks for reporting the issue with a repro, and then following up with a solution!  <3
